homeassistant/components/google_assistant/trait.py

```diff
@@ -649,8 +649,10 @@ def sync_attributes(self):
 
     def query_attributes(self):
         """Return EnergyStorage query attributes."""
-        battery_level = self.state.attributes.get(ATTR_BATTERY_LEVEL) or 0
-        if battery_level == 100:
+        battery_level = self.state.attributes.get(ATTR_BATTERY_LEVEL)
+        if battery_level is None:
+            return {}
```

Comment:
This is not a correct fix, as the battery level is now none, which means we won't be sending other states.

Reply:
descriptiveCapacityRemaining is a required state so all or nothing.

https://developers.google.com/assistant/smarthome/traits/energystorage

Reply:
Ah right! That makes sense :)

Thanks!